jobs/build/ocp4/build.groovy

```diff
@@ -340,11 +347,29 @@ def stageBuildImages() {
             images:build
             --repo-type ${signing_mode}
             """
+
+        def includedCount = commonlib.parseList(buildPlan.imagesIncluded).size()
+        def excludedCount = commonlib.parseList(buildPlan.imagesExcluded).size()
+        def massRebuild = (
+            includedCount && buildPlan.activeImageCount < includedCount * 2 ||  // includes more than half
+            excludedCount && buildPlan.activeImageCount > excludedCount * 2 ||  // excludes less than half
+            !(includedCount || excludedCount)  // rebuilding everything
+        )
+        if (massRebuild) currentBuild.description += "Mass image rebuild (more than half) - invoking serializing semaphore<br/>"
```

Comment:
Think `description` generally is too long already, the most important parts fall off the visible screen. Let's not use description as log line :P 

Think it'd suffice with `(mass)` in description after `4.11`.

Reply:
since the title says `[200 images]` (or whatever) it's already pretty obvious which is a mass rebuild usually, so this was more of a logging statement (esp. for the dry run). That's how I think of the description anyway, as sort of a condensed summary of the build...

In a real run of this, I guess the real question we'd want answered when looking at it is "why is this build taking forever" and the console log would make that pretty evident since it would say "waiting to get a lock" so maybe there's no need for any sort of notice.